src/wallet/bdb.cpp

```diff
@@ -292,11 +292,10 @@ BerkeleyBatch::SafeDbt::operator Dbt*()
     return &m_dbt;
 }
 
-bool BerkeleyBatch::VerifyEnvironment(const fs::path& file_path, bilingual_str& errorStr)
+bool BerkeleyDatabase::VerifyNotInUse(bilingual_str& errorStr)
```

Comment:
In commit "walletdb: Combine VerifyDatabaseFile and VerifyEnvironment" (cba81c3272405332cbb2f3b9c6b2c828c1869d16)

Would call this OpenEnvironment and make it private. Earlier suggestion to add a VerifyNotInUse method https://github.com/bitcoin/bitcoin/pull/19324#discussion_r442532065 was to add a new method wallettool could call, not rename this method in a way which I think is misleading about what it does and how it's used outside of wallettool.

Reply:
I've reverted this back to the single `Verify` and dropped this method entirely as its only usage has also been removed.